packages/server/graphql/types/ReflectPrompt.ts

```diff
@@ -26,25 +26,29 @@ export const customPhaseItemFields = () => ({
     type: new GraphQLNonNull(GraphQLID),
     description: 'foreign key. use the team field'
   },
+  templateId: {
+    type: new GraphQLNonNull(GraphQLID),
+    description: 'foreign key. use the template field'
+  },
   team: {
     type: Team,
-    description: 'The team that owns this customPhaseItem',
+    description: 'The team that owns this reflectPrompt',
     resolve: resolveTeam
   },
   updatedAt: {
     type: new GraphQLNonNull(GraphQLISO8601Type)
   }
 })
 
-const CustomPhaseItem = new GraphQLInterfaceType({
```

Comment:
same deprecation here. whenever we change the GQL schema we wanna do it in at least 2 phases:
- add something new & deprecate the old
- remove the deprecated thing (optional)

that way it gives the old clients time to upgrade.

Reply:
which schema should i export? the CustomPhaseItem or the ReflectPrompt?